src/CosmosDBExperiences.ts

```diff
@@ -5,31 +5,18 @@
 
 import { DatabaseAccount } from 'azure-arm-cosmosdb/lib/models';
 import { IAzureQuickPickItem } from 'vscode-azureextensionui';
+import { API, DBAccountKind, Experience } from './experience';
 
-export enum API {
-    MongoDB = 'MongoDB',
-    Graph = 'Graph',
-    Table = 'Table',
-    Core = 'Core'
-}
-
-export enum DBAccountKind {
-    MongoDB = 'MongoDB',
-    GlobalDocumentDB = 'GlobalDocumentDB'
-}
-
-export type CapabilityName = 'EnableGremlin' | 'EnableTable';
-
-export function getExperienceFromApi(api: API): Experience {
+export function getExperienceFromApi_cosmosdb(api: API): Experience {
     let info = experiencesMap.get(api);
     if (!info) {
         info = { api: api, shortName: api, longName: api, kind: DBAccountKind.GlobalDocumentDB, tag: api };
     }
     return info;
 }
 
-export function getExperienceLabel(account: DatabaseAccount): string {
-    const experience: Experience | undefined = tryGetExperience(account);
+export function getExperienceLabel_cosmosdb(account: DatabaseAccount): string {
```

Comment:
I've never seen a function name with underscores like `getExperienceLabel_cosmosdb`. We usually stick to just camelCase. In this example, I probably would've done getCosmosDBExperienceLabel

Reply:
I fixed it with the code in the master, expect the file name is CosmosDBExperiences and removed PostgresExperiences given an earlier suggestion.  Is required to have method names also specific to cosmosdb or can we keep them as is?